addon/helpers/sort-by.js

```diff
@@ -7,7 +7,11 @@ function normalizeToBoolean(val) {
   }
 
   if (typeof val === 'number') {
-    return val > 0 ? false : true;
+    if (val > 0) {
+      return false;
+    } else if (val < 0) {
+      return true;
+    }
```

Comment:
We shouldn't swap if `val === 0`, thus the need for this refactor

Reply:
Ah! Yes! This is what I could NOT seem to get my head around and fix. Yay!